validator/testcases/javascript/traverser.py

```diff
@@ -83,6 +85,22 @@ def run(self, data):
             raise
         self._debug("END>>")
 
+        for line in sorted(self.complexity.keys()):
```

Comment:
After in-person discussion, it's so the warnings are displayed in the correct order (the key is the line number)